go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/OctopusDeploy/terraform-provider-octopusdeploy
 
-go 1.22
+go 1.22.7
```

Comment:
Ignore all this, I've got my local go-octopusdeploy repo connected up using a go workspace, and it caused some weird ambigious module error. The only way I could get rid of it was to reference `google.golang.org/genproto` directly instead of the sub modules, and this brought in a bunch of other stuff, including a min go bump. I'm hoping once the changes in the go-octopusdeploy library land this won't be a thing.